### ISSUE TITLE
Building test suite 'QueryT' for reflex-0.6 fails

### ISSUE THREAD

**Issue by tshinohara** [reflex-frp/reflex#293]
```
$ nix-build -A ghc8_4.reflex release.nix
...
Preprocessing test suite 'QueryT' for reflex-0.6..
Building test suite 'QueryT' for reflex-0.6..
[1 of 5] Compiling Reflex.TestPlan  ( test/Reflex/TestPlan.hs, dist/build/QueryT/QueryT-tmp/Reflex/TestPlan.o )
[2 of 5] Compiling Reflex.Plan.Reflex ( test/Reflex/Plan/Reflex.hs, dist/build/QueryT/QueryT-tmp/Reflex/Plan/Reflex.o )
[3 of 5] Compiling Reflex.Plan.Pure ( test/Reflex/Plan/Pure.hs, dist/build/QueryT/QueryT-tmp/Reflex/Plan/Pure.o )
[4 of 5] Compiling Test.Run         ( test/Test/Run.hs, dist/build/QueryT/QueryT-tmp/Test/Run.o )
[5 of 5] Compiling Main             ( test/QueryT.hs, dist/build/QueryT/QueryT-tmp/Main.o )

test/QueryT.hs:84:21: error:
    Not in scope: ‘AMap.keys’
    Perhaps you meant ‘Map.keys’ (imported from Data.Map)
    Module ‘Data.AppendMap’ does not export ‘keys’.
   |
84 |   let qDyn = head . AMap.keys . unSelector <$> incrementalToDynamic q
   |                     ^^^^^^^^^

test/QueryT.hs:101:50: error:
    Not in scope: ‘AMap.singleton’
    Perhaps you meant ‘Map.singleton’ (imported from Data.Map)
    Module ‘Data.AppendMap’ does not export ‘singleton’.
    |
101 |         queryDyn $ zipDynWith (\x y -> Selector (AMap.singleton (x :: Int) y)) n $ pure $ MyQuery $ SelectedCount 1
    |                                                  ^^^^^^^^^^^^^^

test/QueryT.hs:116:46: error:
    Not in scope: ‘AMap.singleton’
    Perhaps you meant ‘Map.singleton’ (imported from Data.Map)
    Module ‘Data.AppendMap’ does not export ‘singleton’.
    |
116 |     queryDyn $ zipDynWith (\x y -> Selector (AMap.singleton (x :: Int) y)) n $ pure $ MyQuery $ SelectedCount 1
    |                                              ^^^^^^^^^^^^^^

test/QueryT.hs:130:46: error:
    Not in scope: ‘AMap.singleton’
    Perhaps you meant ‘Map.singleton’ (imported from Data.Map)
    Module ‘Data.AppendMap’ does not export ‘singleton’.
    |
130 |     queryDyn $ zipDynWith (\x y -> Selector (AMap.singleton (x :: Int) y)) n $ pure $ MyQuery $ SelectedCount 1
    |                                              ^^^^^^^^^^^^^^
```

**Comment by ali-abrar** [comment]
Thanks for reporting this. I'll have a look today.

**Comment by ali-abrar** [comment]
Fixed on develop now (and released as 0.6.1). Thanks!
